src/platform/user/selectors.js

```diff
@@ -26,23 +25,12 @@ export const selectAvailableServices = state => selectProfile(state)?.services;
 
 export const selectPatientFacilities = state =>
```

Comment:
This entire selector should be removed. It won't work correctly without the hard-coded data, and it should not be called anywhere. The same is true for:
`selectIsCernerOnlyPatient`
`selectIsCernerPatient`
`selectCernerRxFacilities`
`selectCernerMessagingFacilities`
`selectCernerAppointmentsFacilities`
`selectCernerMedicalRecordsFacilities`
`selectCernerTestResultsFacilities`

HOWEVER, it looks like consuming apps are still using at least one of these. We probably should not commit changes that remove these selectors until we are certain it is okay to do so.

Reply:
Consuming apps removed all usage except one which is covered in this PR. Done 👍